openauto/Service/ServiceFactory.cpp

```diff
@@ -218,6 +236,16 @@ void ServiceFactory::resize()
     }
 #endif
 }
+void ServiceFactory::setAndroidAutoInterface(IAndroidAutoInterface* aa_interface){
```

Comment:
we'd also want to set aa interface for nav service here too right? else its just going to be null there

Reply:
Yes, I'm pretty sure you're correct. Will double check

Reply:
Corrected this to update both nav and media info